### PR TITLE
feat: added cart functionality

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Files to check 
src/pages/cart-page/cart.js
src/context/wishlistContext.jsx
src/components/CartCard.jsx
src/components/product-card/ProductCard.jsx

### CODE REVIEW

File: src/components/cart-card/CartCard.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export const CartCard = ({ _id,brand, title, discountedPrice, actualPrice, discountoff, srcimg, rating}) => {
+    const { addToWishList , wishlist, removeFromCart} = useWishList();
+
+  return (
+    <>
+        <div className="card-horizontal">
+
+            <div className="card-horizontal-container">
+                <div className="horizonal-image">
+                    <img src={ srcimg } alt="shoe image" className="card-horizontal-img" />
```
Comment (sanjitsarkar) [inline src/components/cart-card/CartCard.jsx:14]: **Suggestion:** Try to use underscore or dash  or camelCase in variable consisting of multiple word like here src_img or src-img or srcImg.

**Review by sanjitsarkar — APPROVED** [review]
Everything looks okay and good to merge

**Review by rutu14 — APPROVED** [review]
It is good to merge

**Comment by santhoshpemmaka** [comment]
The code looks good. 👍 Good to merge

**Review by santhoshpemmaka — APPROVED** [review]

**Pull request merged**